generators/entity-server/templates/src/main/java/package/service/dto/EntityDTO.java.ejs

```diff
@@ -62,12 +62,10 @@ import javax.persistence.Lob;
 import <%=packageName%>.domain.enumeration.<%= fields[idx].fieldType %>;
 <%_ } } _%>
 
-<%_ if (typeof javadoc == 'undefined') { _%>
 /**
  * A DTO for the <%= entityClass %> entity.
```

Comment:
This description seems more appropriate to describe this class than the comment from the JDL for the entity.
Otherwise the comment from the JDL would be on both the entity domain and the DTO class.